build.sh

```diff
@@ -23,7 +23,7 @@ sudo apt-get install -y \
 
 sudo mkdir /usr/local/php7
 
-git clone https://github.com/php/php-src.git
+git clone -b PHP-7.0.0 --depth=1 https://github.com/php/php-src.git
 cd php-src
 git checkout PHP-7.0.0
```

Comment:
I personally don't delete the `php-src` directory so every time the build script bumps the PHP version it just simply fetches the new changes. It could be 5 minutes during the first build but only a few seconds during all the subsequent builds.